lib/jms/connection.rb

```diff
@@ -178,11 +177,20 @@ def initialize(params = {})
       connection_factory = nil
       factory = options.delete(:factory)
       if factory
+        # If factory check if oracle is needed.
+        if (factory.include? 'AQjmsFactory')
```

Comment:
Shouldn't this be changed to 'OracleAQConnectionFactory', because this is the name of the factory implemented in oracle_a_q_connection_factory.rb?